packages/workspace/src/atom.ts

```diff
@@ -175,6 +176,7 @@ const fetchMetadata: FetchMetadata = async (get, { signal }) => {
             version: WorkspaceVersion.DatabaseV3,
           }))
         );
+        Adapter.Events['service:start']?.();
```

Comment:
shall we do polling every a few minutes?

Reply:
No, socket.io will handle everything when the connection established